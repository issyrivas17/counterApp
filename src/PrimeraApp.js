   import React from 'react';
   import PropTypes from 'prop-types'; 
   
   /*  import React, { Fragment } from 'react';  */

    const PrimeraApp = ({saludo}) => {
     
        return <>
        <h1>{saludo}</h1>
           {/*  <pre>{JSON.stringify(saludo,null,3)}</pre>  */}
            <p>Mi primera aplicacion web con React</p>
                                                         
        </>;
        } 
        
        PrimeraApp.protoTypes ={
            saludo: PropTypes.string.isRequired
        }
        export default PrimeraApp;  