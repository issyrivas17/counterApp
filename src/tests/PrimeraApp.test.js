import React from 'react';
import { shallow } from "@wojtekmaj/enzyme-adapter-react-17";
import PrimeraApp from "../PrimeraApp"; 
import '@testing-library/jest-dom'

describe('Pruebas en <PrimeraApp/>', () => { 
/*  test('Debe mostrar el mensaje de "Hola,soy ismenia"', () => {
     const saludo = 'Hola,soy ismenia'; 
     const {getByText}= render(<PrimeraApp saludo ={saludo}/>)

    expect(getByText(saludo) ).toBeInTheDocument(); 
 })
 
     */ 

test('Debe de mostrar <PrimeraApp/> correctamente', () => {
    
    const saludo= 'Hola,soy ismenia'; 
const wrapper=shallow(<PrimeraApp saludo={saludo} />) 

expect (wrapper).toMatchSnapshot(); 



})



});
