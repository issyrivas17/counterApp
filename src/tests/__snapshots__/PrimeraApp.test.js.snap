// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp/> Debe de mostrar <PrimeraApp/> correctamente 1`] = `
<Fragment>
  <h1>
    Hola,soy ismenia
  </h1>
  <p>
    Mi primera app web con react
  </p>
</Fragment>
`;
